src/main/cpp/DragonVision/DragonPhotonCam.h

```diff
@@ -0,0 +1,50 @@
+//====================================================================================================================================================
+/// Copyright 2024 Lake Orion Robotics FIRST Team 302
+///
+/// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+/// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+/// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+///
+/// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+///
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+/// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+/// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+/// OR OTHER DEALINGS IN THE SOFTWARE.
+//====================================================================================================================================================
+
+#include "photon/PhotonCamera.h"
+
+#include "DragonVision/DragonCamera.h"
+
+class DragonPhotonCam : public DragonCamera
+{
+public:
+    DragonPhotonCam(std::string name,                       /// <I> - camera name
+                    DragonCamera::PIPELINE initialPipeline, /// <I> - pipeline the camera starts in
+                    units::length::inch_t mountingXOffset,  /// <I> - mounting forward (relative to robot) offset of the limelight
+                    units::length::inch_t mountingYOffset,  /// <I> - mounting horizontal (left/right of robot) from the middle of the robot
+                    units::length::inch_t mountingZOffset,  /// <I> mounting offset up/down
+                    units::angle::degree_t pitch,           /// <I> - Pitch of limelight
+                    units::angle::degree_t yaw,             /// <I> - Yaw of limelight
+                    units::angle::degree_t roll);           /// <I> - Roll of limelight
+
+    /// Need to add DragonCamera overrides
+
+    units::angle::degree_t GetTargetHorizontalOffset() const;
```

Comment:
shouldn't override be added for all these functions

Reply:
No, I talked to Mr. Wits about it and that causes some errors and weird behavior when you override a virtual function.